src/element/textElement.ts

```diff
@@ -673,6 +673,29 @@ export const shouldAllowVerticalAlign = (
   });
 };
 
+export const suppportsHorizontalAlign = (
+  selectedElements: NonDeletedExcalidrawElement[],
+) => {
+  return selectedElements.some((element) => {
+    const hasBoundContainer = isBoundToContainer(element);
+    if (hasBoundContainer) {
+      const container = getContainerElement(element);
+      if (isTextElement(element) && isArrowElement(container)) {
+        return false;
+      }
+      return true;
+    }
+    const boundTextElement = getBoundTextElement(element);
+    if (boundTextElement) {
+      if (isArrowElement(element)) {
+        return false;
+      }
+      return true;
+    }
```

Comment:
what is this for? Seems same as the check above it.

Reply:
since we fetch selected Elements including the bound text (https://github.com/excalidraw/excalidraw/blob/master/src/scene/selection.ts#L81) hence this block can be removed